code-studio/src/js/react-only.js

```diff
@@ -6,3 +6,4 @@
 
 window.React = require('react');
 window.ReactDOM = require('react-dom');
+window.Radium = require('radium');
```

Comment:
No, the name seems appropriate.  We load `React` and `ReactDOM` and `Radium` but in all it's sort of one framework and its plugins.